src/components/Articles/SingleArticleComponent.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export default ({ article }) => (
+  <div className="item" key={article.id}>
+    <div className="image">
+      <img src="https://placeimg.com/640/480/nature" alt="andela" />
+    </div>
+    <div className="top aligned content">
+      <a className="header" href="article.html">{article.title}</a>
```

Comment:
To avoid the page reloading, please use `Link` from react-router in place of the anchor tags.